src/fragile-goods/lib/logic.ergo

```diff
@@ -39,19 +39,36 @@ contract FragileGoods over FragileGoodsClause {
       if contract.deliveryLimitDuration.unit = "seconds"
       then
         define variable duration = momentDiffSeconds(finishTime, request.startTime);
-        if (duration) > contract.deliveryLimitDuration.amount
-        then return new PayOut{
-		      amount: new org.accordproject.money.MonetaryAmount{
-						doubleValue: amount - contract.lateDeliveryPenalty.doubleValue,
-						currencyCode: currency
-					}
-				}
-        else return new PayOut{ 
-		      amount: new org.accordproject.money.MonetaryAmount{
-						doubleValue: amount,
-						currencyCode: currency
-					}
-				}
+        if (duration) > contract.deliveryLimitDuration.amount then
+          define variable monetaryAmount = new org.accordproject.money.MonetaryAmount {
+            doubleValue: amount - contract.lateDeliveryPenalty.doubleValue,
+            currencyCode: currency
+          };
+          emit new org.accordproject.cicero.runtime.PaymentObligation {
+            contract: "NOT_SET",
```

Comment:
@jeromesimeon Note the hard-coded contract ID here. This is required as this is a clause template, but we don't yet allow independent declarations of clauses without contracts.